cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,14 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: {{ .ImageRepository }}/busybox-{{ .Arch }}
```

Comment:
While it would be great to get this merged, I'm inclined to await https://github.com/kubernetes/kubernetes/pull/46597 to land and then I can remove the init container.